dom.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="dom.css" />
+    <title>Portfolio page</title>
+      <style>
+        /* Style for the experiences section */
+        #experiences {
+            border: 1px solid #ccc;
+            padding: 10px;
+            max-height: 300px; /* Add a max height and enable scrolling */
+            /* Enable vertical scrollbar  */
+             overflow-y: auto;     
+        }
+
+        /* Style for the form (initially hidden) */
+        #experienceForm {
+           display: none; 
+        }
+    </style>
+    <header>
+      <!-- The Header section -->
+      <div class="header">
+          <a href="http://127.0.0.1:5500/signup.html">Login!</a>
```

Comment:
still this ip address is being used. you need to implement this without providing the ip-address

Reply:
will be done in next PR (this file is not part of this PR)